resources/wikia/modules/uuid.js

```diff
@@ -0,0 +1,8 @@
+define('uuid', function () {
+	return function () {
+		return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
+			var r = Math.random() * 16 | 0, v = c === 'x' ? r : (r & 0x3 | 0x8);
```

Comment:
that's some hardcore magic 👌

Reply:
it is copy-pasted from generating pageview unique id, @bkoval is working on sharing it